app/component/CityBikeStopNearYou.js

```diff
@@ -23,6 +22,7 @@ const CityBikeStopNearYou = ({ stop, relay, currentTime, currentMode }) => {
       );
     }
   }, [currentTime]);
+  // todo
```

Comment:
This todo is not required any longer, as it is replaced by custom CityBikeStopNearYou.okc